Sources/MessagesDisplayDataSource.swift

```diff
@@ -26,7 +26,9 @@ import Foundation
 
 public protocol MessagesDisplayDataSource: class, MessagesDataSource {
 
-    func messageColorFor(_ message: MessageType, at indexPath: IndexPath, in messagesCollectionView: MessagesCollectionView) -> UIColor
+    func textColorFor(_ message: MessageType) -> UIColor
+    
+    func backgroundColorFor(_ message: MessageType) -> UIColor
```

Comment:
but what is the reasoning for `in messagesCollectionView: MessagesCollectionView`